packages/server/src/service/pdfForm/util.ts

```diff
@@ -30,6 +30,23 @@ export const splitFullName = (fullName: string) => {
   return [firstName, middleName, lastName, suffix]
 }
 
+export const stripPhoneNumber = (phoneNumber: string) => {
+  // Get rid of characters. split-join acts as 'replaceAll'
+  let strippedNumber = phoneNumber
+    .replace('+', '')
+    .replace('(', '')
+    .replace(')', '')
+    .split('-')
```

Comment:
I would just do replace('-', '') and don't need a join

Reply:
This is to basically 'replaceAll': https://stackoverflow.com/questions/1144783/how-to-replace-all-occurrences-of-a-string

I was surprised to find that there's no native 'replaceAll' function -- unless I'm completely overlooking something.

Reply:
Good point -- the 2nd answer /pattern/g is how you do global replace